src/serializer.ts

```diff
@@ -3,23 +3,22 @@ import { bsv, num2bin } from './utils';
 /*
  * a varint serializer into Script ASM
  */
-
+const Script = bsv.Script;
 const BN = bsv.crypto.BN;
 // number of bytes to denote state length after serialization, exclusing varint prefix
-const STATE_LEN = 2;
+const STATE_LEN = 4;
```

Comment:
2 bytes can encode upto 2^15 bytes long, which is more than enough for most contracts. Let's keep it to save some space.